typescript/testsuite/main.ts

```diff
@@ -0,0 +1,26 @@
+import { ExampleTestsuiteConfigurator } from "./execution_impl/example_testsuite_configurator";
+import { TestSuiteExecutor } from "kurtosis-testsuite-api-lib";
+import * as log from "loglevel";
+
+const SUCCESS_EXIT_CODE: number = 0;
+const FAILURE_EXIT_CODE: number = 1;
+
+// >>>>>>>>>>>>>>>>>>> REPLACE WITH YOUR OWN CONFIGURATOR <<<<<<<<<<<<<<<<<<<<<<<<
+const configurator: ExampleTestsuiteConfigurator = new ExampleTestsuiteConfigurator();
+// >>>>>>>>>>>>>>>>>>> REPLACE WITH YOUR OWN CONFIGURATOR <<<<<<<<<<<<<<<<<<<<<<<<
+
+const suiteExecutor: TestSuiteExecutor = new TestSuiteExecutor(configurator);
+let exitCode: number = SUCCESS_EXIT_CODE;
```

Comment:
This isn't right - this is supposed to be inside the `.then` callback, because it's only used inside it